### PR TITLE
Create Love Calc Command

### PR DESCRIPTION
**Pull request by ghost**
New command to calculate how much you love a member

### CODE REVIEW

File: commands/fun/lovecalc.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      name: "lovecalc",
+      description: "Find out how much you love anoyher user",
+      category: "Fun",
+      usage: "lovecalc @<user>",
+      cost: 10,
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    const data = await get(`https://love-calculator.p.mashape.com/getPercentage?fname=${message.member.displayName}&sname=${message.mentions.members.first().displayName}`).set("X-Mashape-Key", "MASHAPE API KEY HERE");
```
Comment (AndrewJWin) [inline commands/fun/lovecalc.js]: Here's a quick issue: What if message.mentions.first() is not valid?

**Review by AndrewJWin — CHANGES_REQUESTED** [review]
L 17, C 108. Missing a semicolon. Also, change ``"K then, die forever alone"`` to something less.. *Harsh*, she's tsundere but not ***that*** tsundere.

**Comment by OGNova** [comment]
Dunno if this has been addressed, but typo in the description, `anoyher` should be `another`.

**Comment by auguwu** [comment]
Instead of `MASHUP KEY HERE`, maybe use `this.client.config.<where_keys_are>`?

File: commands/fun/lovecalc.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      name: "lovecalc",
+      description: "Find out how much you love another user",
+      category: "Fun",
+      usage: "lovecalc @<user>",
+      cost: 10,
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    if (!message.mentions.members.first()) return message.response(undefined, "Ba-Baka! How will I tell you, how much you love someone. If I don't know who!"); //Response Can Be Refined 😂
```
Comment (kyranet) [inline commands/fun/lovecalc.js]: `if (!message.mentions.members.size)`

**Review by kyranet — CHANGES_REQUESTED** [review]
Just one little thing

**Review by kyranet — APPROVED** [review]

**Pull request merged**